src/npm/doNpmPublish.ts

```diff
@@ -1,9 +1,25 @@
+import getOptionalFlagArgument from "../utils/getOptionalFlagArgument.js";
 import spawn from "../utils/spawn.js";
 import appendCmdIfWindows from "./utils/appendCmdIfWindows.js";
 import npmLogger from "./utils/npmLogger.js";
 
-export default async function (PATH: string) {
-  await spawn(appendCmdIfWindows`npm`, ["publish"], npmLogger, {
+interface NpmOptions {
+  /**
+   * If you ask npm to install a package and don't tell it a specific version, then it will install the specified tag.
```

Comment:
This phrase is hurting my brain a little (to be fair, i'm barely awake 😴 )

version !== tag? 🤔

Reply:
Yeah, I copied-pasta the doc from npm, but it's gibberish.

Simplified it.

But no, version !== tag in npm.

For npm, you could think of version as Git commit SHA: they're unique etc
And tag, well, it's the same: it's just a label that you attach to a commit/version